doc/users-guide/users-guide-utilities-examples.adoc

```diff
@@ -0,0 +1,17 @@
+== Utilities and examples
+
+=== PcapNg capture
+If compiled using `--enable-pcapng-support` ODP will offer packet capturing
+functionality in PcapNg format. If /var/run/odp directory exists prior to
+launching the application ODP will create as many fifos as the NIC queues.
```

Comment:
...as many fifos as there are NIC queues. Alternately, "will create a fifo for each NIC queue".